.github/workflows/validate_and_publish_release.yml

```diff
@@ -25,6 +25,12 @@ jobs:
         with:
           release_id: ${{ github.event.inputs.release_id }}
 
+      - name: Checking out repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.WRITE_ACCESS_TOKEN }}
+          ref: "${{ env.tag_name }}"
```

Comment:
We don't have `tag_name` in the env here. The build_release pipeline is passing through `tag_name` as the ref to this pipeline, so the default ref here is already the one we want.

```suggestion
```

Reply:
Done.